terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java

```diff
@@ -356,7 +356,7 @@ public void test06_Query_Display() {
         // output 06_05 Test
         assertThat(
                 driver.findElement(
-                        By.xpath("//a[contains(@href, \"?page=1&size=10&age=0&countries=&dateOfBirth=&main=false&name=%253Cscript%253Ealert('XSS%2520Attack')%253C/script%253E\")]"))
+                        By.xpath("//a[contains(@href, \"?page=1&size=10&age=0&_countries=&_dateOfBirth=&main=false&name=%3Cscript%3Ealert('XSS%20Attack')%3C/script%3E\")]"))
                         .getText(), is("2"));

```

Comment:
In current implementation, URL encoding is duplicate(= bad URL encoding).
Bad URL encoding implementation see [here](https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/master/terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/el/beanQueryOutput.jsp#L55-L56).

Bad URL encoding flow is follows:

Target value : `"<script>..."`
　↓
1st of URL encoding by `q:query` : `"%3Cscript%3E..."` (`<` -> `%3C`, `>` -> `%3E` )
　↓
2nd of URL encoding by `queryTmpl` : `"%253Cscript%253E..."` ( `%` -> `%25` )

**2nd URL encoding(`%` -> `%25` ) is unnecessary.** We should specified a query string that was converted using the `f:query` into the `criteriaQuery` attribute of `<t:pagination>` tag.